.github/workflows/pr-build.yml

```
@@ -67,6 +69,10 @@ jobs:
         with:
           arguments: build integrationTests --stacktrace -PenableCoverage=true -PlocalDocker=true
 
+      - name: Create checksum
+        run: |
+          echo ORIG_CHECKSUM="$(sha256sum /otel-auto-instrumentation/javaagent.jar | cut -d' ' -f1)" >> "$GITHUB_ENV"
```

```suggestion
          echo "ORIG_CHECKSUM=$(sha256sum /otel-auto-instrumentation/javaagent.jar | cut -d' ' -f1)" >> $GITHUB_ENV
```
I think this would set the env in right format